go.mod

```diff
@@ -0,0 +1,44 @@
+module k8s.io/client-go
+
+go 1.16
+
+retract (
+	// pre-module versions are obsolete
+	v12.0.0+incompatible
+	// pre-module versions are obsolete
+	v11.0.0+incompatible
+	// pre-module versions are obsolete
+	v10.0.0+incompatible
+	// pre-module versions are obsolete
+	v9.0.0+incompatible
```

Comment:
We also have a `v9.0.0-invalid` - https://github.com/kubernetes/client-go/tree/v9.0.0-invalid

Should that be included here?

Reply:
excellent catch... I don't know how I missed that